common/pom.xml

```diff
@@ -68,5 +68,10 @@
       <groupId>org.roaringbitmap</groupId>
       <artifactId>RoaringBitmap</artifactId>
     </dependency>
+    <dependency>
+      <groupId>net.jpountz.lz4</groupId>
+      <artifactId>lz4</artifactId>
+      <version>1.3.0</version>
```

Comment:
we manage the version in the parent pom file.

Reply:
ok